pkg/owner/manager_test.go

```diff
@@ -78,6 +79,17 @@ func (ti *testInfo) Close(t *testing.T) {
 	ti.cluster.Terminate(t)
 }
 
+type listener struct {
+	val atomic.Bool
```

Comment:
we should not use atomic variables here because the real listeners are not concurrent safe, like l.scheduler is a normal field. If we don't use atomic variables herem, race detector in unit tests can report error when `ownerManager` breaks the no concurrent calling property.